zero_bin/leader/src/client.rs

```diff
@@ -46,6 +46,25 @@ pub(crate) async fn client_main(
         ),
     ));
 
+    if let BlockInterval::FollowFrom { start_block, .. } = block_interval.clone() {
```

Comment:
@Nashtare this should take care if we pass checkpoint_block_number in leader binary